language-support/hs/bindings/README.md

```diff
@@ -21,21 +21,55 @@ You can find some usage examples
-published to Hackage at the moment.
-
-To make things a bit easier there is a `build_packages.sh` script in
-this directory that you can run as follows: `./build_packages.sh
-TARGET_DIR`.  This will build the tarballs for all cabal packages and
-put them in `TARGET_DIR`. If `TARGET_DIR` does not already contain a
-`cabal.project` file, it will also create a `cabal.project` file that
-references the created tarballs so all dependencies should be resolved
-properly. You will also need to install gRPC 1.22, see
-https://github.com/grpc/grpc/blob/master/BUILDING.md for installation
-instructions. If you install it in a non-standard location, you need
-to adjust extra-lib-dirs and extra-include-dirs for
-grpc-haskell-core. The default `cabal.project` file contains an
-example of how to do this.
-
-The main package for the ledger bindings is called `daml-ledger`.
+
+# Using these bindings **outside of this repository**
+
+*Note: These instructions document the current process. We do plan make it simpler!*
+
+These bindings can be build as a standalone Haskell package `daml-ledger` using `stack`. Currently, `stack` references the code directly in the `daml` repo. The only annoyance is that part of the Haskell code for `daml-ledger` is generated from `.proto` files, and this generation step must be performed using the `bazel` build. We plan to simplify this step by directly providing `daml-ledger` as a package on hackage.
+
+As well as `stack`, the only other prerequisite is to have `grpc` installed.
+We are currently using gRPC version `1.22.1`. To install `grpc`  requires building `grpc` from source (really!). See detailed instructions here: https://github.com/grpc/grpc/blob/master/BUILDING.md.
+
+Note, the `grpc` build instructions warn against doing a global install:
+
+> *"WARNING: After installing with make install there is no easy way to uninstall, which can cause issues if you later want to remove the grpc and/or protobuf installation or upgrade to a newer version."*
+
+If you follow this advice, then you must adjust `extra-lib-dirs` in your `stack.yaml` config. This is explicated in the detailed instructions below.
+
+
+## Download and build grpc at version 1.22.1
+
+    cd ~/github
+    git clone -b $(curl -L https://grpc.io/release) https://github.com/grpc/grpc
+    cd grpc
+    git checkout v1.22.1
+    git submodule update --init
+    make
+
+## Clone daml repo; force Haskell code generation
+
+    cd ~/github
+    git clone https://github.com/digital-asset/daml.git
+    cd daml
+    direnv allow
+    bazel build ledger-api/grpc-definitions:google-protobuf-haskellpb-sources ledger-api/grpc-definitions:google-rpc-haskellpb-sources ledger-api/grpc-definitions:ledger-api-haskellpb-sources ledger-api/grpc-definitions:ledger-api-haskellpb-sources-admin ledger-api/grpc-definitions:ledger-api-haskellpb-sources-testing
+
+
+## Write a DAML Ledger App in Haskell (or copy one!)
+
+    cd /tmp
```

Comment:
```suggestion
    cd $(mktemp -d)
```

Reply:
I played around with using `$(mktemp -d)` in the instuctions... but in the end it seemed to make things less clear, so I went back to `/tmp'

But overall, the instructions are simpler than before.

The instructions no longer have a step to patch the `stack.yaml` copied from the `nim` example with the location of the `daml-ledger` package... because the original file assumes the package has been exported to `/tmp`, and that matches where the instructions export it to!